drivers/flash/flash_stm32.c

```diff
@@ -383,6 +389,9 @@ static int stm32_flash_init(const struct device *dev)
 	LOG_DBG("Flash initialized. BS: %zu",
 		flash_stm32_parameters.write_block_size);
 
+	/* Check Flash configuration */
+	flash_stm32_check_configuration();
```

Comment:
Shouldn't we check function's output here?

Reply:
Yes, indeed.
Done